workflow/artifacts/hdfs/hdfs.go

```diff
@@ -7,8 +7,8 @@ import (
 	"path/filepath"
 
 	"github.com/argoproj/pkg/file"
-	"gopkg.in/jcmturner/gokrb5.v5/credentials"
-	"gopkg.in/jcmturner/gokrb5.v5/keytab"
+	credentialsv8 "github.com/jcmturner/gokrb5/v8/credentials"
+	keytabv8 "github.com/jcmturner/gokrb5/v8/keytab"
```

Comment:
nit: previously the version of the package where not part of the name, what is the advantage of adding it?

Reply:
No certain reason tbh, just kept it at that moment. 
I can happily get rid of the `v8`s in the naming :) 

PS: Apologies for the late reply, my github notifications clutter is a mess to deal with :P